components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OASParserUtil.java

```diff
@@ -1467,4 +1467,40 @@ public static String getAuthorizationHeaderFromSwagger(Map<String, Object> exten
         return authorizationHeader == null ? null : authorizationHeader.toString();
     }
 
+    /**
+     * This method returns extension of application security types related to micro-gw
+     *
+     * @param extensions Map<String, Object>
+     * @return String
```

Comment:
@return type wrong. And there is no use of this doc comment if you don't add a description on what it returns.

Reply:
Fixed with c1f846270025eb67fcd8e92c102a8539e1e699a3